resources/definitions/anycubic_mega_zero.def.json

```diff
@@ -0,0 +1,98 @@
+{
+    "version": 2,
+    "name": "Anycubic Mega Zero",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "kad",
+        "manufacturer": "Anycubic",
+        "file_formats": "text/x-gcode",
+        "platform": "anycubic_mega_zero_platform.stl",
+        "has_materials": true,
+        "has_machine_quality": true,
+        "preferred_quality_type": "normal",
+        "preferred_material": "generic_pla",
+        "machine_extruder_trains":
+        {
+            "0": "anycubic_mega_zero_extruder_0"
+        }
+    },
+
+    "overrides":
+    {
+        "machine_name":
+        {
+            "default_value": "Anycubic Mega Zero"
+        },
+        "machine_heated_bed":
+        {
+            "default_value": false
+        },
+        "machine_width":
+        {
+            "default_value": 224
```

Comment:
On the [spec page](https://www.anycubic.com/products/mega-zero) they mention that the build plate is 220x220. Was this size a deliberate deviation from that?

Reply:
updated to 220x220, with minor start code tweak.

Reply:
Well if it's safe to use a bigger volume size, that's fine too. I was just making sure that it was intentional.